src/helpers/ConfigHelper.ts

```diff
@@ -84,8 +113,8 @@ export class ConfigHelper {
       ];
 
       const slidedownConfigKeys = Object.keys(slidedownConfig);
-      for (let i=0; i<version1Keys.length; i++) {
-        for (let j=0; j<slidedownConfigKeys.length; j++) {
+      for (let i = 0; i < version1Keys.length; i++) {
+        for (let j = 0; j < slidedownConfigKeys.length; j++) {
```

Comment:
Clean up: Instead of nested `for` loop you could you `hasOwnProperty` or `in`.
https://stackoverflow.com/q/455338/1244574